### ISSUE TITLE
Invalid URL u'https://:/api/oauth2/token/': No host supplied

### ISSUE THREAD

**Issue by JeffreyDevloo** [openvstorage/framework#794]
# Problem description

Cannot extend a vpool beyond the master node (node where the vpool was spawned)
# Possible root of the problem

User management?
# Possible solution

None
### Additional information

Used a Hyperconverged setup:
- 3 nodes
- Each node has 3 Backend disks
- Running unstable 2.7.2-rev.3849.a0ed527-1
### Stacktrace

```
2016-08-09 14:34:28 97800 +0200 - ovs-node1 - 2073/140163966129984 - celery/celery.worker.job - 167 - DEBUG - Task accepted: ovs.storagerouter.add_vpool[b3aff932-687b-481c-afca-fa7379b5bdaf] pid:2091
2016-08-09 14:34:34 28900 +0200 - ovs-node1 - 2073/140163966129984 - celery/celery.worker.job - 168 - ERROR - Task ovs.storagerouter.add_vpool[b3aff932-687b-481c-afca-fa7379b5bdaf] raised unexpected: InvalidURL("Invalid URL u'https://:/api/oauth2/token/': No host supplied",)
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/lib/storagerouter.py", line 424, in add_vpool
    backend_dict = ovs_client.get('/alba/backends/{0}/'.format(backend_guid), params={'contents': 'metadata_information,name,usages,presets'})
  File "/opt/OpenvStorage/ovs/extensions/api/client.py", line 180, in get
    return self._call(api=api, params=params, function=requests.get)
  File "/opt/OpenvStorage/ovs/extensions/api/client.py", line 158, in _call
    headers, url = self._prepare(params=params)
  File "/opt/OpenvStorage/ovs/extensions/api/client.py", line 109, in _prepare
    self._connect()
  File "/opt/OpenvStorage/ovs/extensions/api/client.py", line 88, in _connect
    verify=self._verify)
  File "/usr/local/lib/python2.7/dist-packages/requests/api.py", line 107, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/api.py", line 53, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 454, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 388, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 293, in prepare
    self.prepare_url(url, params)
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 356, in prepare_url
    raise InvalidURL("Invalid URL %r: No host supplied" % url)
InvalidURL: Invalid URL u'https://:/api/oauth2/token/': No host supplied
```

---

**Comment by khenderick** [comment]
I'll give this some priority since this blocks quite soms tests

**Comment by khenderick** [comment]
Fixed by #796, available in openvstorage-2.7.2-rev.3857.71749e3

**Comment by JeffreyDevloo** [comment]
Verified, extending works for unstable 2.7.2-rev.3857.71749e3-1 .
